GCC/CORTEX_A53_64-bit_UltraScale_MPSoC/port.c

```diff
@@ -780,6 +781,130 @@ void vIRQHandler( uint32_t ulICCIAR )
     uxCoreInIsr[xCoreId]--;
 }
 
+#define portRTOS_LOCK_COUNT     (2u)
+#define portMAX_CORE_COUNT      configNUMBER_OF_CORES
```

Comment:
Already modified

Reply:
I don't think I was clear. What is the point of defining `portMAX_CORE_COUNT` if it will only be `configNUMBER_OF_CORES`? Why not just use `configNUMBER_OF_CORES` everywhere you've used `portMAX_CORE_COUNT`?